aten/src/THC/generic/THCTensorMath.h

```diff
@@ -4,15 +4,17 @@
 
 THC_API void THCTensor_(fill)(THCState *state, THCTensor *self, scalar_t value);
 THC_API void THCTensor_(zero)(THCState *state, THCTensor *self);
-
-THC_API ptrdiff_t THCTensor_(numel)(THCState *state, THCTensor *t);
 THC_API void THCTensor_(cat)(THCState *state, THCTensor *result, THCTensor *ta, THCTensor *tb, int dimension);
```

Comment:
are we testing cat with bool?

Reply:
we do, that't how i've found out about - 
https://github.com/pytorch/pytorch/blob/master/test/test_torch.py#L9329-L9330, this calls _cat_.